@xen-orchestra/lite/src/stories/web-core/ui-card.story.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[prop('direction').preset('column').enum('column', 'row').required().widget()]"
```

Comment:
If using the `horizontal` prop as suggested in my comment in the component:
```suggestion
    :params="[prop('horizontal').bool().widget()]"
```